scripts/fm

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /bin/sh
```

Comment:
I believe `bash` was better here. What if someones default shell is `fish` syntax would not match